src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <container class="stars stars--0">
```

Comment:
`container` is not a standart html element, unless there is a special reason, use `div` here